iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_sender_data.hpp

```diff
@@ -27,7 +27,7 @@ namespace iox
 {
 namespace popo
 {
-template <typename ChunkDistributorDataType>
+template <uint32_t MaxChunksPerSender, typename ChunkDistributorDataType>
```

Comment:
And here maybe something like MaxChunksAllocatedInParallel

Reply:
What about "MaxChunksAllocatedSimultaneously"